uv.lock

```diff
@@ -1008,7 +1008,7 @@ wheels = [
 
 [[package]]
 name = "dbt-core"
-version = "1.9.4"
+version = "1.9.3"
```

Comment:
it's weird to see a version downgrade - do you know why that happened?

Reply:
i think its just because I needed to run `uv sync -U`! `multidict` upgraded in that diff, and now downgrades after syncing. Im not immediately sure how the uv precommit let me get away with that